src/potentiometer-element.ts

```diff
@@ -0,0 +1,269 @@
+
+  render() {
+    return html`
+      <style>
+        #${this.id} {
+          transform-origin: -120.168198px -140.5804px;
+          transform: rotate(var(--knob-angle, 0deg));
+        }
+      </style>
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        width="20mm"
+        height="20mm"
+        viewBox="0 0 20 20"
+        @mousedown=${this.down}
+        @mousemove=${this.move}
+        @mouseup=${this.up}
+        @touchstart=${this.down}
+        @touchmove=${this.move}
+        @touchend=${this.up}
+      >
+        <g transform="translate(-6.7420057,-6.6693388)">
+          <g transform="matrix(0.15090139,0,0,0.15090139,34.862601,35.84314)">
+            <rect
+              ry="8.1522207"
+              y="-192.35448"
+              x="-185.37505"
+              height="128.89323"
+              width="129.11356"
+              style="opacity:1;fill:#045881;fill-opacity:1;stroke:#045881;stroke-width:1.95151615;stroke-linecap:butt;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
+            />
+            <rect
+              y="-188.67798"
+              x="-150.76115"
+              height="12.407617"
+              width="60.234947"
+              style="opacity:1;fill:#ccdae3;fill-opacity:1;stroke:none;stroke-width:2.26042461;stroke-linecap:butt;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
+            />
+            <ellipse
+              ry="49.255947"
+              rx="48.154297"
+              cy="-139.14471"
+              cx="-120.70818"
+              style="opacity:1;fill:#e4e8eb;fill-opacity:1;stroke:none;stroke-width:1.97907138;stroke-linecap:butt;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
+            />
+            <rect
+              y="-80.647011"
+              x="-142.63103"
+              height="13.660479"
+              width="42.74408"
+              style="opacity:1;fill:#000000;fill-opacity:0;stroke:#ffffff;stroke-width:1.95151615;stroke-linecap:butt;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
+            />
+            <text
+              y="-83.291"
+              x="-145.16521"
+              style="font-style:normal;font-weight:normal;font-size:6.4255476px;line-height:1.25;font-family:sans-serif;letter-spacing:0px;word-spacing:0px;fill:#ffffff;fill-opacity:1;stroke:none;stroke-width:0.34422573"
+              xml:space="preserve"
+            >
+              <tspan
+                style="font-size:6.4255476px;fill:#ffffff;stroke-width:0.34422573"
+                y="-83.291"
+                x="-174.16521"
+              >
+                GND
+              </tspan>
+            </text>
+            <text
+              y="-83.155312"
+              x="-128.39781"
+              style="font-style:normal;font-weight:normal;font-size:6.42554712px;line-height:1.25;font-family:sans-serif;letter-spacing:0px;word-spacing:0px;fill:#ffffff;fill-opacity:1;stroke:none;stroke-width:0.34422573"
+              xml:space="preserve"
```

Comment:
Similar to the graphics items, most of the styles here are redundant, as the `font-style` and `font-weight` are already the default.

I'd move all the font related styles (font-size, line-height, font-family, etc.) to the top SVG element, so that they won't have to be repeated for each `text` element. 

And the font size can be shortened to just `6.4`, without all the trailing decimals... (Inkscape just loves them!)

Reply:
Done